include/cynq/enums.hpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * See LICENSE for more information about licensing
+ *
+ * Copyright 2023
+ * Author: Luis G. Leon-Vega <luis.leon@ieee.org>
+ *         Diego Arturo Avila Torres <diego.avila@uned.cr>
+ *
+ */
+#pragma once
+
+
+struct Status {
+/**
```

Comment:
There is a wrong indentation. Have you used `clang-format -i --style=google <FILE>`?